torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1326,23 +1332,22 @@ def sample_inputs_addmv(op_info, device, dtype, requires_grad, **kwargs):
                                  ((), (S, M), (M,), 0.2, 0.6, True),
                                  )
 
-    cases = test_cases + test_cases_with_broadcast
-    sample_inputs = []
-    for input_args in cases:
-        args = (make_tensor(input_args[0], device, dtype,
-                            low=None, high=None,
-                            requires_grad=requires_grad),
-                make_tensor(input_args[1], device, dtype,
-                            low=None, high=None,
-                            requires_grad=requires_grad),
-                make_tensor(input_args[2], device, dtype,
-                            low=None, high=None,
-                            requires_grad=requires_grad))
-        alpha, beta = input_args[3], input_args[4]
-        broadcasts_input = input_args[5]
-        sample_inputs.append(SampleInput(args[0], args=(args[1], args[2]), kwargs=dict(beta=beta, alpha=alpha),
-                                         broadcasts_input=broadcasts_input))
-    return tuple(sample_inputs)
+    if dtype.is_complex:
+        test_cases_complex = (((S,), (S, M), (M, ), (1, 2), (3, 4), False),
+                              ((S,), (S, M), (M, ), (1.2, 2.3), (3.4, 4), False),
+                              )
+        cases = test_cases + test_cases_with_broadcast + test_cases_complex
```

Comment:
Earlier `addmv` wasn't generating complex cases for beta and alpha values, thought it would be worth testing - hence this if condition.